### PR TITLE
Cisco firepower e streamer cef

### PR DESCRIPTION
**Pull request by skhademcis**
Fixes #

## Proposed Changes

  -  Updated Description and Application name in the MD

### CODE REVIEW

**Comment by v-jayakal** [comment]
@skhademcis, Can you provide changes to the existing files in the Sentinel Repo to track the changes and the best way to identify and modify the changes needed. 
Please use https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json to modify the connector changes. 
Thanks.

**Comment by v-jayakal** [comment]
@skhademcis, Please resolve conflicts and make changes to the existing files in the repo for modifications.

**Comment by v-jayakal** [comment]
Hey @skhademcis , Please make changes to the existing files instead of adding them resulting in conflicts. Can you please modify the files? Let us know if you need more information. Thanks.

File: DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json [hunk 1]
```diff
@@ -39,8 +39,7 @@
         }
     ],
     "availability": {
-        "status": 1,
-        "isPreview": true
+        "status": 1
```
Comment (v-admahe) [inline DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json:42]: isPreview tag is missing. Please add the isPreview tag.

Thanks,
Aditi Maheshwari

File: DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json [hunk 2]
```diff
@@ -86,7 +85,7 @@
                                     "PrimaryKey"
                                 ],
                                 "label": "Run the following command to install and apply the CEF collector:",
-                                "value": "sudo wget -O cef_installer.py https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_installer.py&&sudo python cef_installer.py {0} {1}"
+                                "value": "sudo wget https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_installer.py&&sudo python cef_installer.py {0} {1}"
```
Comment (v-admahe) [inline DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json:88]: -O cef_installer.py is missing. Please refer the sample template https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json

Thanks,
Aditi Maheshwari

File: DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json [hunk 3]
```diff
@@ -95,43 +94,43 @@
...
+            "title": "3. Validate connection",
+            "description": "Follow the instructions to validate your connectivity:\n\nOpen Log Analytics to check if the logs are received using the CommonSecurityLog schema.\n\n>It may take about 20 minutes until the connection streams data to your workspace.\n\nIf the logs are not received, run the following connectivity validation script:\n\n> 1. Make sure that you have Python on your machine using the following command: python -version\n\n>2. You must have elevated permissions (sudo) on your machine",
+            "instructions": [
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "WorkspaceId"
+                        ],
+                        "label": "Run the following command to validate your connectivity:",
+                        "value": "sudo wget https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_troubleshoot.py&&sudo python cef_troubleshoot.py  {0}"
```
Comment (v-admahe) [inline DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json:128]: -O cef_troubleshoot.py is missing. Please refer the sample template https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json

Thanks,
Aditi Maheshwari

File: Logos/cisco-logo-72px.svg [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
-            <path d="M47,7.63829787 C47,6.81914894 46.25,6 45.5,6 C44.625,6 44,6.70212766 44,7.63829787 L44,15.3617021 C44,16.2978723 44.625,17 45.5,17 C46.375,17 47,16.2978723 47,15.3617021 L47,7.63829787" id="eee67ecc-f43c-4348-a1c0-d20d3d15934b"/>
-            <path d="M56,1.85483871 C56,0.865591398 55.375,0 54.5,0 C53.625,0 53,0.865591398 53,1.85483871 L53,21.1451613 C53,22.1344086 53.75,23 54.5,23 C55.375,23 56,22.1344086 56,21.1451613 L56,1.85483871" id="0215bbe7-b01e-4b17-97b7-d4eaec1a1228"/>
-            <path d="M64,7.63829787 C64,6.81914894 63.375,6 62.5,6 C61.625,6 61,6.70212766 61,7.63829787 L61,15.3617021 C61,16.2978723 61.625,17 62.5,17 C63.375,17 64,16.2978723 64,15.3617021 L64,7.63829787" id="d5468f7e-db62-42a1-9c6d-d0e2193209e0"/>
-            <path d="M72,12.4716981 C72,11.6792453 71.25,11 70.5,11 C69.625,11 69,11.6792453 69,12.4716981 L69,15.5283019 C69,16.3207547 69.625,17 70.5,17 C71.375,17 72,16.3207547 72,15.5283019 L72,12.4716981" id="7a0dcd37-2cc9-4d43-95f0-28ccd746441b"/>
-        </g>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="72px" height="72px" viewBox="0 0 72 72">
```
Comment (v-admahe) [inline Logos/cisco-logo-72px.svg:2]: Please add xmlns="http://www.w3.org/2000/svg" in svg tag.

Thanks,
Aditi Maheshwari

**Review by v-admahe — CHANGES_REQUESTED** [review]

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**